doc/api/deprecations.md

```diff
@@ -189,15 +189,6 @@ Type: Runtime
 
 The `Intl.v8BreakIterator` is deprecated and will be removed or replaced soon.
 
-<a id="DEP0018"></a>
-### DEP0018: Unhandled promise rejections
-
-Type: Runtime
-
-Unhandled promise rejections are deprecated. In the future, promise rejections
-that are not handled will terminate the Node.js process with a non-zero exit
-code.
-
```

Comment:
Shouldn't it be marked EOL instead of being removed?

Reply:
Yes. It should be marked End-of-life